enterprise-registry/github-app/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: docs
+title: Register GitHub Application
+category: registry
+sub_category: setup
+forkurl: https://github.com/coreos/docs/blob/master/enterprise-registry/github-app/index.md
+weight: 5
+---
+
+# Creating an OAuth Application in GitHub
+
```

Comment:
Can we add an overview of what this gets you, as a user? I get all of those Github docs confused...I'm sure someone else will as well.